### ISSUE TITLE
Unable to play captions and subtitles (v8 testathon) on Safari

### ISSUE THREAD

**Issue by keithgee** [DSpace/dspace-angular#2961]
**Describe the bug**
Captions and subtitles won't play back for videos on sandbox.dspace.org, which is hosting the testathon.

I believe my filenames are compliant according to [documentation for the prior version](https://wiki.lyrasis.org/display/DSDOC7x/User+Interface+Configuration#UserInterfaceConfiguration-Uploadingvideocaptioningfiles), but it's possible that I'm forgetting something; we've occasionally had problems getting captions to work with individual items with prior versions.

**To Reproduce**
1. Find an item with a video supported by the media viewer, and a .vtt caption/subtitle file with the correct naming conventions. For testing, I submitted a new item, uploading both files at on item creation. This [item in the sandbox ](https://sandbox.dspace.org/items/74f6de86-3511-4ad1-a174-4873410c279c) may be deleted by the time you see this report, but I took the files from [an item in our repository](https://vtechworks.lib.vt.edu/items/c292e667-7245-485c-8925-39c2b85e96a5).
2. Start video playback
3. Enable captions (in this example English) in the usual fashion. In my case, there's a disclosure arrow on the right side of the playback controls. (I tested this on a Mac with Safari). The captions won't display, even after speech starts.

**Expected behavior**
I expected the text from the captions to be visible in, or immediately next to the media playback window.

**Comment by DanGastardelli** [comment]
Good afternoon @tdonohue!

We at Neki-it did a current test with a video with subtitles in Safari on MacOS and it worked perfectly (using DSpace 8).

**Comment by tdonohue** [comment]
Thanks @DanGastardelli for verifying this functionality works for you! In that case, I'll close this ticket as unreproducible.  (I'm not able to reproduce this issue in other browsers either)

If anyone else discovers a way to reproduce this in the future, we can always reopen this ticket as necessary.
